demo-shell/proxy.conf.js

```diff
@@ -1,14 +1,29 @@
 require('dotenv').config();
 
-const { getDeployedAppsProxy, getShareProxy, getApsProxy } = require('./proxy-helpers');
-
-const legacyHost = process.env.PROXY_HOST_ADF;
-const cloudHost = process.env.CLOUD_PROXY_HOST_ADF;
-const cloudApps = process.env.APP_CONFIG_APPS_DEPLOYED;
-const apsHost = process.env.PROXY_HOST_ADF;
-
+var PROXY_HOST_ADF = process.env.PROXY_HOST_ADF;
+console.log('PROXY_HOST_ADF' + PROXY_HOST_ADF);
 module.exports = {
-    ...getShareProxy(legacyHost),
-    ...getApsProxy(apsHost),
-    ...getDeployedAppsProxy(cloudHost, cloudApps)
+    "/alfresco": {
```

Comment:
Why o you need this? I'm not sure this change is correct.

Reply:
I'm afraid this might cause problems for the others.

Reply:
you are right, thank you! :)